src/styles/block/calendar.scss

```diff
@@ -0,0 +1,44 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: ($day-size * 7) + ($gap-size * 6) + ($padding-size * 2);
+  padding: $padding-size;
+  gap: $gap-size;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $day-size;
+    height: $day-size;
+    font-size: 30px;
+    background-color: #eee;
+    border: 1px solid black;
+    transition: 0.5s transform;
```

Comment:
check the syntax and list all transition properties, it is better for performance